tests/unit/badge/badge.renderBadge.spec.js

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
```

Comment:
Not to brag about this, but the `Calendar`'s spec is a good example when it comes to use a single file to test something.

However if you look at `tests/unit/calendar/` you will see 2 files (_.normal._ and _.range._). _Why?_ you may ask... Well, I decided to split them on the 'type' of calendar, since it changes the expectations and/or main behavior/rendering process. In that case it does simplify the reading and maintainability.
Could they be in the same file? Yes... Should they? _Maybe_, but I did a 'virtual' separation.

So, @maximuk you're free to do a separation on those functions, though it is my strong belief that you should only split them in case you cannot _touch_ the functions via normal render :)

Last but not the least, if you're wondering why the .specs. do not use the `matchSnapshot()` methods, it is mainly due to the fact that there are `Date`s involved in the rendering... Not that easy to maintain as the snapshots would keep changing :) Usually it fits, this is a special case 😄 

Hope I was clear.